OpenSim/Common/Scale.h

```diff
@@ -96,15 +96,17 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Scale, Object);
 
     void getScaleFactors(SimTK::Vec3& aScaleFactors) const;
     SimTK::Vec3& getScaleFactors() { return _scaleFactors; }
+#ifndef SWIG    
     void getScaleFactors(double rScaleFactors[]){   // A variant that uses basic types for use by GUI
         getScaleFactors(SimTK::Vec3::updAs(rScaleFactors));
     }
-
+#endif
```

Comment:
This method was introduced for GUI use, I'd remove it rather than idndef SWIG it.